backend/src/main/resources/application.yaml

```diff
@@ -63,7 +81,7 @@ management:
 okta:
   oauth2:
     issuer: https://hhs-prime.okta.com/oauth2/default
-    client-id: ${OKTA_OAUTH2_CLIENT_ID}
+    client-id: ${OKTA_OAUTH2_CLIENT_ID:MISSING}
```

Comment:
why was the `MISSING` needed here?

Reply:
When you try to run this locally without missing it throws an error `Could not resolve placeholder 'OKTA_OAUTH2_CLIENT_ID' in value "${OKTA_OAUTH2_CLIENT_ID}"`

Reply:
Would adding that property in the application-dev.yaml with the :MISSING to do an specific overwrite make sense?

Reply:
Yeah that would make sense. I also noticed this change can be made in our current spring boot version so pr #6901 will handle all of these changes.